NOT A REAL CHANGE: accidental whitespace / reindentation drift

NOT A REAL CHANGE - do not re-apply after updating mupdf.

Reindentation that a stray editor or formatter pass left behind in the vendored
tree. No hunk here changes behaviour.

One exception worth fixing rather than carrying forward: in
platform/x11/x11_main.c a newline was inserted into the middle of the getopt
option string literal. That file is not built on Windows, which is why nobody
noticed.

This patch exists only so that applying the whole series reproduces ext/mupdf
byte for byte, which is what makes the series verifiable.
---
diff --git a/platform/x11/x11_main.c b/platform/x11/x11_main.c
index eb6fc09..8648e62 100644
--- a/platform/x11/x11_main.c
+++ b/platform/x11/x11_main.c
@@ -934,7 +934,8 @@ int main(int argc, char **argv)
 
 	pdfapp_init(ctx, &gapp);
 
-	while ((c = fz_getopt(argc, argv, "Ip:r:A:C:W:H:S:U:Xb:c:v")) != -1)
+	while ((c = fz_getopt(argc, argv, "Ip:r:A:C:W:H:S:U:Xb:c:
+	v")) != -1)
 	{
 		switch (c)
 		{
diff --git a/source/fitz/load-jpx.c b/source/fitz/load-jpx.c
index 19f54e2..5ca7bba 100644
--- a/source/fitz/load-jpx.c
+++ b/source/fitz/load-jpx.c
@@ -477,13 +477,13 @@ jpx_read_image(fz_context *ctx, fz_jpxd *state, const unsigned char *data, size_
 
 	if (!onlymeta)
 	{
-		if (!opj_decode(codec, stream, jpx))
-		{
-			opj_stream_destroy(stream);
-			opj_destroy_codec(codec);
-			opj_image_destroy(jpx);
-			fz_throw(ctx, FZ_ERROR_LIBRARY, "Failed to decode JPX image");
-		}
+	if (!opj_decode(codec, stream, jpx))
+	{
+		opj_stream_destroy(stream);
+		opj_destroy_codec(codec);
+		opj_image_destroy(jpx);
+		fz_throw(ctx, FZ_ERROR_LIBRARY, "Failed to decode JPX image");
+	}
 	}
 
 	opj_stream_destroy(stream);
@@ -543,14 +543,14 @@ jpx_read_image(fz_context *ctx, fz_jpxd *state, const unsigned char *data, size_
 	if (!onlymeta)
 	{
 		for (k = 0; k < numcomps; ++k)
+	{
+		if (!jpx->comps[k].data)
 		{
-			if (!jpx->comps[k].data)
-			{
-				opj_image_destroy(jpx);
-				fz_throw(ctx, FZ_ERROR_FORMAT, "image components are missing data");
-			}
+			opj_image_destroy(jpx);
+			fz_throw(ctx, FZ_ERROR_FORMAT, "image components are missing data");
 		}
 	}
+	}
 
 	w = state->width = jpx->x1 - jpx->x0;
 	h = state->height = jpx->y1 - jpx->y0;
diff --git a/source/fitz/stext-search.c b/source/fitz/stext-search.c
index ce08148..17b02e6 100644
--- a/source/fitz/stext-search.c
+++ b/source/fitz/stext-search.c
@@ -976,10 +976,10 @@ flush_transform_cache(fz_context *ctx, char *output, size_t *index, fz_text_tran
 		{
 			int i_ccc = ucdn_get_combining_class(cache->cache[i]);
 			for (j = i+1; j < n; j++)
-			{
+	{
 				int j_ccc = ucdn_get_combining_class(cache->cache[j]);
 				if (i_ccc > j_ccc)
-				{
+		{
 					int t = cache->cache[j];
 					cache->cache[j] = cache->cache[i];
 					cache->cache[i] = t;
@@ -1013,7 +1013,7 @@ flush_transform_cache(fz_context *ctx, char *output, size_t *index, fz_text_tran
 			}
 		}
 		n = i+1;
-	}
+}
 
 	/* Now actually do the output */
 	len = 0;
@@ -1113,10 +1113,10 @@ transform_char(fz_context *ctx, char *output, size_t *index, int c, size_t pos,
 		c = fz_toupper(c);
 
 	if (c == '\n' && (transform & (FZ_TEXT_TRANSFORM_KEEP_LINES | FZ_TEXT_TRANSFORM_KEEP_PARAGRAPHS)))
-	{
+		{
 		/* Don't strip \n if we might need it. */
-	}
-	else
+		}
+		else
 		c = canon(c);
 
 	/* Squash runs of spaces. */
@@ -1136,7 +1136,7 @@ transform_char(fz_context *ctx, char *output, size_t *index, int c, size_t pos,
 		cache->pos[cache->len] = pos;
 		cache->len++;
 		return 0;
-	}
+		}
 	/* Alternatively, if we've got a combining character, then we want to store that in the cache. */
 	if (ccc > 0)
 	{
@@ -1195,7 +1195,7 @@ do_transform(fz_context *ctx, fz_text_transform transform, char *output, size_t
 		*index = pos;
 	}
 	if (output)
-	{
+			{
 		output += len;
 		*output = 0;
 	}
@@ -1206,25 +1206,25 @@ do_transform(fz_context *ctx, fz_text_transform transform, char *output, size_t
 
 static char *
 transform_text_with_index(fz_context *ctx, fz_text_transform transform, const char *input, size_t **indexp)
-{
+				{
 	char *output;
 	size_t len = do_transform(ctx, transform, NULL, NULL, input);
 
 	output = fz_malloc(ctx, len);
 	fz_try(ctx)
-	{
+					{
 		*indexp = fz_malloc_array(ctx, len+1, size_t);
 	}
 	fz_catch(ctx)
-	{
+						{
 		fz_free(ctx, output);
 		fz_rethrow(ctx);
-	}
+						}
 
 	(void)do_transform(ctx, transform, output, *indexp, input);
 
 	return output;
-}
+					}
 
 static char *
 transform_text_without_index(fz_context *ctx, fz_text_transform transform, const char *input)
diff --git a/source/html/epub-doc.c b/source/html/epub-doc.c
index 67cea90..91ff023 100644
--- a/source/html/epub-doc.c
+++ b/source/html/epub-doc.c
@@ -69,6 +69,7 @@ typedef struct
 	 * to the most recently used html block here, thus
 	 * ensuring that the stored copy won't be evicted. */
 	fz_html *most_recent_html;
+
 } epub_document;
 
 struct epub_chapter
diff --git a/source/html/html-parse.c b/source/html/html-parse.c
index ff03121..8a440f6 100644
--- a/source/html/html-parse.c
+++ b/source/html/html-parse.c
@@ -642,33 +642,33 @@ static void generate_text(fz_context *ctx, fz_html_box *box, const char *text, i
 				const char *p = mark;
 				n = fz_chartorune(&c, p);
 				while (p < text)
-				{
+			{
 					p += n;
 					if (fz_isletter_or_apos(c))
-					{
+				{
 						while (p < text)
 						{
 							n = fz_chartorune(&c, p);
 							if (!fz_isletter_or_apos(c))
 								break;
 							p += n;
-						}
+				}
 						generate_text_run_with_hyphens(ctx, box, flow, mark, p, lang, hyph, g);
 					}
 					else
-					{
+				{
 						while (p < text)
-						{
+					{
 							n = fz_chartorune(&c, p);
 							if (fz_isletter_or_apos(c))
 								break;
 							p += n;
 						}
 						generate_text_run(ctx, box, flow, mark, p, lang, g);
-					}
+						}
 					mark = p;
+					}
 				}
-			}
 			else
 			{
 				generate_text_run(ctx, box, flow, mark, text, lang, g);
@@ -1435,16 +1435,16 @@ static void gen2_tag(fz_context *ctx, struct genstate *g, fz_html_box *root_box,
 
 	if (style->direction == FZ_BIDI_UNSET)
 	{
-		dir_att = fz_xml_att(node, "dir");
-		if (dir_att)
-		{
-			if (!strcmp(dir_att, "auto"))
-				g->markup_dir = FZ_BIDI_NEUTRAL;
-			else if (!strcmp(dir_att, "rtl"))
-				g->markup_dir = FZ_BIDI_RTL;
-			else if (!strcmp(dir_att, "ltr"))
-				g->markup_dir = FZ_BIDI_LTR;
-			else
+	dir_att = fz_xml_att(node, "dir");
+	if (dir_att)
+	{
+		if (!strcmp(dir_att, "auto"))
+			g->markup_dir = FZ_BIDI_NEUTRAL;
+		else if (!strcmp(dir_att, "rtl"))
+			g->markup_dir = FZ_BIDI_RTL;
+		else if (!strcmp(dir_att, "ltr"))
+			g->markup_dir = FZ_BIDI_LTR;
+		else
 				g->markup_dir = FZ_BIDI_LTR;
 		}
 	}
@@ -2729,9 +2729,9 @@ fz_debug_html_box(fz_context *ctx, fz_html_box *box, int level)
 			fz_debug_css_number(level, "height", box->style->height);
 			if (box->u.block.margin[0] != 0 || box->u.block.margin[1] != 0 || box->u.block.margin[2] != 0 || box->u.block.margin[3] != 0)
 			{
-				indent(level+1);
-				printf(">margin=(%g %g %g %g)\n", box->u.block.margin[0], box->u.block.margin[1], box->u.block.margin[2], box->u.block.margin[3]);
-			}
+			indent(level+1);
+			printf(">margin=(%g %g %g %g)\n", box->u.block.margin[0], box->u.block.margin[1], box->u.block.margin[2], box->u.block.margin[3]);
+		}
 			if (box->u.block.border[0] != 0 || box->u.block.border[1] != 0 || box->u.block.border[2] != 0 || box->u.block.border[3] != 0)
 			{
 				indent(level+1);
diff --git a/source/pdf/pdf-js.c b/source/pdf/pdf-js.c
index 7a40623..3a12553 100644
--- a/source/pdf/pdf-js.c
+++ b/source/pdf/pdf-js.c
@@ -552,13 +552,13 @@ static void doc_resetForm(js_State *J)
 		fz_try(ctx)
 		{
 			n = pdf_array_len(ctx, form);
-			for (i = 0; i < n; i++)
+		for (i = 0; i < n; i++)
 				pdf_field_reset(ctx, js->doc, pdf_array_get(ctx, form, i));
 		}
-		fz_catch(ctx)
-			rethrow(js);
+			fz_catch(ctx)
+				rethrow(js);
+		}
 	}
-}
 
 static void doc_print(js_State *J)
 {
diff --git a/source/pdf/pdf-link.c b/source/pdf/pdf-link.c
index 72744ce..90d0c4a 100644
--- a/source/pdf/pdf-link.c
+++ b/source/pdf/pdf-link.c
@@ -138,6 +138,7 @@ populate_destination(fz_context *ctx, pdf_document *doc, pdf_obj *dest, int is_r
 		p = fz_transform_point_xy(arg1v, arg2v, ctm);
 		destination->x = arg1 ? p.x : NAN;
 		destination->y = arg2 ? p.y : NAN;
+		/* SumatraPDF */
 		destination->zoom = arg3 ? (arg3v > 0 ? (arg3v * 100) : NAN) : NAN;
 		break;
 	case FZ_LINK_DEST_FIT_R:
@@ -265,17 +266,17 @@ get_file_stream_and_name(fz_context *ctx, pdf_obj *fs, pdf_obj **namep)
 	 * spec, but we'd rather find the embedded file than
 	 * not. */
 	if (any_name && !file)
-	{
+{
 		name = any_name;
 		file = pdf_dict_get(ctx, ef, PDF_NAME(UF));
 		if (file == NULL)
-			file = pdf_dict_get(ctx, ef, PDF_NAME(F));
+		file = pdf_dict_get(ctx, ef, PDF_NAME(F));
 		if (file == NULL)
-			file = pdf_dict_get(ctx, ef, PDF_NAME(Unix));
+		file = pdf_dict_get(ctx, ef, PDF_NAME(Unix));
 		if (file == NULL)
-			file = pdf_dict_get(ctx, ef, PDF_NAME(DOS));
+		file = pdf_dict_get(ctx, ef, PDF_NAME(DOS));
 		if (file == NULL)
-			file = pdf_dict_get(ctx, ef, PDF_NAME(Mac));
+		file = pdf_dict_get(ctx, ef, PDF_NAME(Mac));
 	}
 
 	if (namep)
diff --git a/source/pdf/pdf-op-run.c b/source/pdf/pdf-op-run.c
index cf448b7..2c98e44 100644
--- a/source/pdf/pdf-op-run.c
+++ b/source/pdf/pdf-op-run.c
@@ -1029,9 +1029,9 @@ pdf_show_path(fz_context *ctx, pdf_run_processor *pr, int doclose, int dofill, i
 					break;
 				if (gstate->fill.alpha != 1)
 					fz_begin_group(ctx, pr->dev, bbox, NULL, 0, 0, FZ_BLEND_NORMAL, gstate->fill.alpha);
-				fz_clip_path(ctx, pr->dev, path, even_odd, gstate->ctm, bbox);
-				gstate = pdf_show_pattern(ctx, pr, gstate->fill.pattern, gstate->fill.gstate_num, bbox, PDF_FILL);
-				fz_pop_clip(ctx, pr->dev);
+					fz_clip_path(ctx, pr->dev, path, even_odd, gstate->ctm, bbox);
+					gstate = pdf_show_pattern(ctx, pr, gstate->fill.pattern, gstate->fill.gstate_num, bbox, PDF_FILL);
+					fz_pop_clip(ctx, pr->dev);
 				if (gstate->fill.alpha != 1)
 					fz_end_group(ctx, pr->dev);
 				break;
@@ -1064,9 +1064,9 @@ pdf_show_path(fz_context *ctx, pdf_run_processor *pr, int doclose, int dofill, i
 					break;
 				if (gstate->stroke.alpha != 1)
 					fz_begin_group(ctx, pr->dev, bbox, NULL, 0, 0, FZ_BLEND_NORMAL, gstate->stroke.alpha);
-				fz_clip_stroke_path(ctx, pr->dev, path, gstate->stroke_state, gstate->ctm, bbox);
-				gstate = pdf_show_pattern(ctx, pr, gstate->stroke.pattern, gstate->stroke.gstate_num, bbox, PDF_STROKE);
-				fz_pop_clip(ctx, pr->dev);
+					fz_clip_stroke_path(ctx, pr->dev, path, gstate->stroke_state, gstate->ctm, bbox);
+					gstate = pdf_show_pattern(ctx, pr, gstate->stroke.pattern, gstate->stroke.gstate_num, bbox, PDF_STROKE);
+					fz_pop_clip(ctx, pr->dev);
 				if (gstate->stroke.alpha != 1)
 					fz_end_group(ctx, pr->dev);
 				break;
@@ -1630,9 +1630,9 @@ pdf_show_text(fz_context *ctx, pdf_run_processor *pr, pdf_obj *text)
 				/* Bug 708615: pdf_show_char inside show_string can realloc gstate. */
 				gstate = pr->gstate + pr->gtop;
 				pdf_show_space(ctx, pr, - pdf_to_real(ctx, item) * gstate->text.size * 0.001f);
-			}
 		}
 	}
+	}
 	else if (pdf_is_string(ctx, text))
 	{
 		show_string(ctx, pr, (unsigned char *)pdf_to_str_buf(ctx, text), pdf_to_str_len(ctx, text));
@@ -2013,7 +2013,7 @@ get_lineage(fz_context *ctx, pdf_obj *a, int *lenp)
 		{
 			/* Put a into lineage. */
 			if (max == len)
-			{
+{
 				max *= 2;
 				if (max == 0)
 					max = 32;
@@ -2026,8 +2026,8 @@ get_lineage(fz_context *ctx, pdf_obj *a, int *lenp)
 				break;
 
 			if (a == line[len>>1])
-				fz_throw(ctx, FZ_ERROR_FORMAT, "cycle in structure tree");
-		}
+			fz_throw(ctx, FZ_ERROR_FORMAT, "cycle in structure tree");
+	}
 	}
 	fz_catch(ctx)
 	{
@@ -2245,13 +2245,13 @@ push_marked_content(fz_context *ctx, pdf_run_processor *proc, const char *tagstr
 		/* No point in layer processing unnecessarily */
 		if (proc->process_layers)
 		{
-			/* Start any optional content layers. */
-			if (pdf_name_eq(ctx, tag, PDF_NAME(OC)))
-				begin_oc(ctx, proc, val, NULL);
+		/* Start any optional content layers. */
+		if (pdf_name_eq(ctx, tag, PDF_NAME(OC)))
+			begin_oc(ctx, proc, val, NULL);
 
-			/* Special handling for common non-spec extension. */
-			if (pdf_name_eq(ctx, tag, PDF_NAME(Layer)))
-				begin_layer(ctx, proc, val);
+		/* Special handling for common non-spec extension. */
+		if (pdf_name_eq(ctx, tag, PDF_NAME(Layer)))
+			begin_layer(ctx, proc, val);
 		}
 
 		/* Structure */
@@ -3308,7 +3308,7 @@ pdf_close_run_processor(fz_context *ctx, pdf_processor *proc)
 	pop_nest_until_empty(ctx, pr);
 
 	if (pr->process_structure)
-		pop_structure_to(ctx, pr, NULL);
+	pop_structure_to(ctx, pr, NULL);
 
 	clear_marked_content(ctx, pr);
 }
diff --git a/source/pdf/pdf-xref.c b/source/pdf/pdf-xref.c
index 2533a28..f8199c8 100644
--- a/source/pdf/pdf-xref.c
+++ b/source/pdf/pdf-xref.c
@@ -2682,7 +2682,7 @@ pdf_load_object(fz_context *ctx, pdf_document *doc, int num)
 		return NULL;
 	entry = pdf_cache_object(ctx, doc, num);
 	if (entry)
-		return pdf_keep_obj(ctx, entry->obj);
+	return pdf_keep_obj(ctx, entry->obj);
 	return NULL;
 }
 
